s3-source-connector/build.gradle.kts

```diff
@@ -67,31 +67,17 @@ dependencies {
   compileOnly(apache.kafka.connect.runtime)
 
   implementation(project(":commons"))
+  implementation(project(":s3-commons"))
   implementation("com.amazonaws:aws-java-sdk-s3:$amazonS3Version")
   implementation("com.amazonaws:aws-java-sdk-sts:$amazonSTSVersion")
 
-  implementation("org.apache.parquet:parquet-hadoop:$parquetVersion")
-  testImplementation("org.apache.parquet:parquet-hadoop:$parquetVersion")
-  integrationTestImplementation("org.apache.parquet:parquet-hadoop:$parquetVersion")
-
-  implementation("org.apache.parquet:parquet-avro:$parquetVersion") {
-    exclude(group = "org.xerial.snappy", module = "snappy-java")
-    exclude(group = "org.slf4j", module = "slf4j-api")
-    exclude(group = "org.apache.avro", module = "avro")
-  }
-  testImplementation("org.apache.parquet:parquet-avro:$parquetVersion") {
-    exclude(group = "org.xerial.snappy", module = "snappy-java")
-    exclude(group = "org.slf4j", module = "slf4j-api")
-    exclude(group = "org.apache.avro", module = "avro")
-  }
-
   implementation(tools.spotbugs.annotations)
   implementation(logginglibs.slf4j)
   implementation(apache.avro)
   implementation(confluent.kafka.connect.avro.converter) {
     exclude(group = "org.apache.kafka", module = "kafka-clients")
   }
-
+  integrationTestImplementation("org.apache.parquet:parquet-hadoop:1.11.2")
```

Comment:
It's not clear to me -- why aren't we using the $parquetVersion here?

Reply:
I missed this, I should have updated the parquetVersion after my test passed instead of leaving it as is. Thanks